Sources/Actions/Admin/News.php

```diff
@@ -223,18 +224,6 @@ function addNewsItem ()
 		'settings' => ['settings', 'admin_forum'],
 	];
 
-	/****************************
-	 * Internal static properties
-	 ****************************/
-
-	/**
-	 * @var self
-	 *
-	 * An instance of this class.
-	 * This is used by the load() method to prevent multiple instantiations.
-	 */
-	protected static News|ACP $obj;
-
```

Comment:
We need to restore these lines for this class, because the object could be of a type other than `self`.